docs/compiling-mono/linux/index.md

```diff
@@ -76,7 +78,7 @@ sudo mkdir $PREFIX
 sudo chown -R `whoami` $PREFIX
 
 # Ensure that all required packages are installed.
-sudo apt-get install git autoconf libtool automake build-essential mono-devel gettext
+sudo apt-get install git autoconf libtool libtool-bin automake build-essential mono-devel gettext
```

Comment:
Sure. Just make sure that `lsb_release` is actually available on the system before trying to use it, as it might not be available on some Debian versions out of the box.